parodos-model-api/src/main/java/com/redhat/parodos/workflow/task/BaseWorkFlowTask.java

```diff
@@ -47,7 +47,7 @@ public void setBeanName(String name) {
 		this.name = name;
 	}
 
-	protected String getParameterValue(WorkContext workContext, String parameterName) throws MissingParameterException {
+	public String getParameterValue(WorkContext workContext, String parameterName) throws MissingParameterException {
```

Comment:
Is there any reason to change visibility for this method?

Reply:
Yes, I need to invoke `getParameterValue()` method in [RestAPIWorkFlowTaskTest](https://github.com/redhat-developer/parodos/blob/92236853d9327c02a3ed096032bcf743c9062e8e/workflow-examples/src/test/java/com/redhat/parodos/examples/simple/RestAPIWorkFlowTaskTest.java#L44) and [SecureAPIGetTestTaskTest](https://github.com/redhat-developer/parodos/blob/92236853d9327c02a3ed096032bcf743c9062e8e/workflow-examples/src/test/java/com/redhat/parodos/examples/simple/SecureAPIGetTestTaskTest.java#L53) so `protected` is not enough anymore.
Does make it sense to you?

Reply:
Oh I see that those tests are in different package.